src/rgw/rgw_d3n_datacache.h

```diff
@@ -41,7 +40,7 @@ struct D3nChunkDataInfo : public LRUObject {
 };
 
 struct D3nCacheAioWriteRequest {
-	std::string oid;
+        std::string oid;
```

Comment:
Knit: Not sure why this indentation change was made but this change should be removed